src/Model/Url.php

```diff
@@ -18,10 +18,8 @@ final class Url implements UrlInterface
 
     private ?float $priority = null;
 
-    /** @var Collection|AlternativeUrlInterface[] */
```

Comment:
I think we should keep these, or is there another way to make sure the IDE knows this collection holds 'AlternativeUrlInterface' objects?